virt/support/monitoring/virt-monitoring-overview.adoc

```diff
@@ -9,7 +9,9 @@ toc::[]
 You can monitor the health of your cluster and virtual machines (VMs) with the following tools:
 
 xref:../../../virt/support/monitoring/virt-running-cluster-checkups.adoc#virt-running-cluster-checkups[{product-title} cluster checkup framework]::
-Check network connectivity and latency by using predefined, automated tests.
+Run automated tests on your cluster with the {product-title} cluster checkup framework to check the following:
```

Comment:
Per ISG, you need a descriptive word for the items in the list after _following_.
For ex:

> check the following conditions:

Reply:
Thank-you! I knew that I needed a description but I could not think of the right word. :-)